crates/bevy_math/src/interpolation/utils.rs

```diff
@@ -0,0 +1,67 @@
+use std::ops::{Add, Div, Mul, Sub};
+
+/// Quake 3 fast inverse sqrt
```

Comment:
I would be in favor of replacing this algorithm with the algorithm from [Walcyzk et al. 2018](https://arxiv.org/abs/1802.06302). It's claimed to be significantly more accurate without additional computation costs. Page 15 has the final algorithm that we would be implementing.

Reply:
I added a TODO note to do this later, make in the next revision